newrelic/resource_newrelic_synthetics_private_location_test.go

```diff
@@ -0,0 +1,110 @@
+//go:build integration
+// +build integration
+
+package newrelic
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/newrelic/newrelic-client-go/pkg/common"
+	"testing"
+)
+
+func TestAccNewRelicSyntheticsPrivateLocation_Basic(t *testing.T) {
+	resourceName := "newrelic_synthetics_private_location.bar"
+	rName := acctest.RandString(5)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckNewRelicSyntheticsPrivateLocationDestroy,
+		Steps: []resource.TestStep{
+			// Test: Create
+			{
+				Config: testAccNewRelicSyntheticsPrivateLocationConfig(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNewRelicSyntheticsPrivateLocationExists(resourceName),
+				),
+			},
+			// Test: Update
+			{
+				Config: testAccNewRelicSyntheticsPrivateLocationConfigUpdated(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNewRelicSyntheticsPrivateLocationExists(resourceName),
+				),
+			},
+			// Test: Import
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"description", "domain_id", "key", "location_id", "verified_script_execution"},
```

Comment:
@mbazhlekova We seem to be ignoring a lot on import, so I'm wondering if this import function has any value for customer. From what I understand we won't get the "domain_id", "key", "location_id" ever if the user imports?

Reply:
@kidk Since we have to read from entity, we don't get a lot of that information back. Import can still be useful since we'll get the guid to use in a monitor.